src/main/java/com/stripe/model/Customer.java

```diff
@@ -13,7 +13,7 @@
 
 public class Customer extends APIResource implements MetadataStore<Customer>, HasId {
 	String id;
-	Integer accountBalance;
+	Long accountBalance;
```

Comment:
@dukehoops By default these fields get a modifier called "package-private" which means that they can only be used internally to the package. It's true that private might be a little better, but at least we aren't leaking an API.